src/components/Checkbox/Checkbox.scss

```diff
@@ -1,5 +1,6 @@
 @import "../../styles/global-css-settings";
```

Comment:
please check this file 


please in the mixin create for example variable  toggle-primary-bg-color: $black-2 and then use with them-prop